content/python/concepts/matplotlib/terms/legend/legend.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: '.legend()'
+Description: 'Places a legend on the axes and access to manipulate parameters of the legend.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Libraries'
+  - 'Matplotlib'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.legend()`** function returns a legend on the axes of a plot or subplot.
```

Comment:
```suggestion
The **`pyplot.legend()`** function returns a legend on the axes of a plot or subplot.
```

I think it's important to include the module that this function came from so it won't be [confused](https://matplotlib.org/stable/search.html?q=legend) with something else in the Matplotlib ecosystem.